tests/functional_test.js

```diff
@@ -232,7 +231,7 @@ describe('functional', function () {
             }
 
             publicURIs.push(uri);
-        });
+	});
```

Comment:
Yeah, was working in windows and I don't have my vim config there to adjust whitespace.